Sources/ProjectSpec/TestTargeReference.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import JSONUtilities
+
+public struct TestableTargetReference: Hashable {
```

Comment:
Instead of introducing `TestableTargetReference` would it be possible to just enhance `TargetReference`. I'm asking because is build targets can also be in local Swift Packages and not just test targets?